### DISCUSSION TITLE
Release 0.7.3-rc?

### DISCUSSION THREAD

**Discussion by Nihisil** [danny-avila/LibreChat#2698] [Feature Requests & Suggestions]
Is there a way to make new release with this workflow: https://github.com/danny-avila/LibreChat/actions/workflows/tag-images.yml ? It is ok that it will include other things pushed since 0.7.2 release.

So, people who are using Docker setup can use latest GPT-4o model and don't need to wait few weeks for new release: https://github.com/danny-avila/LibreChat/pull/2692

Thank you for your amazing project :pray:

**Comment by danny-avila** [comment]
Sounds good to me, will cut a release candidate tomorrow

**Comment by danny-avila** [comment]
https://github.com/danny-avila/LibreChat/releases/tag/v0.7.3-rc

**Reply by Nihisil** [reply]
Thank you!
